docs/walkthroughs/download-artifact-to-device.adoc

```diff
@@ -0,0 +1,59 @@
+[[mobile-client-builds]]
+= Mobile Client Builds
+
+== Open Mobile Client Builds
+To download artifact you need navigate to Overview -> Mobile Clients -> <your-mobile-client>
+
+image::images/mobile-client-overview.png[openshift-overview-tab]
+
+Then go to Builds tab to open mobile client builds
+
+image::images/mobile-client-builds-tab.png[openshift-mobile-client-builds-tab]
+
+== Start Mobile Client Build
+As you see that there are not builds currently available
+When you want to trigger new build press [Start Build] button
+
+After starting build you can see notification that build was started and also
+you see that build was triggered under BUILDS section
+
+image::images/mobile-client-builds-trigger-build.png[openshift-mobile-client-builds-trigger-build]
+
+== Download Mobile Client Artifact
+You need to wait until build is completed and then press [Download] button
+
+image::images/mobile-clients-builds-build-completed.png[openshift-mobile-client-builds-build-completed]
+
+You should see bar code and use any of bar code scanner applications in Google Play for Android or
+or App Store for iPhone. If you have bar code scanner installed then you need to scan
+code and it should open URL you see on screen above
+
+[[mobile-client-manual-installation]]
+= Manual Artifact Installation
```

Comment:
Worth mentioning that they can also use the url that is being displayed in openshift.